src/Carousel.vue

```diff
@@ -813,11 +835,14 @@ export default {
     },
     render() {
       // add extra slides depending on the momemtum speed
-      this.offset +=
-        Math.max(
-          -this.currentPerPage + 1,
-          Math.min(Math.round(this.dragMomentum), this.currentPerPage - 1)
-        ) * this.slideWidth;
+      if (this.rtl) {
```

Comment:
Not sure what's going on here - do we not want this feature with rtl scrolling (I've never seen rtl scrolling before, so this is more informative than critical lol)

Reply:
My bad, I forgot to convert its logic to support RTL

Reply:
Ok cool! Glad I asked 😎